shared/test/metabase/mbql/util_test.cljc

```diff
@@ -362,6 +362,12 @@
     (t/is (= [:not [:contains [:field 1 nil] "ABC" {:case-sensitive false}]]
              (mbql.u/desugar-filter-clause [:does-not-contain [:field 1 nil] "ABC" {:case-sensitive false}])))))
 
+(t/deftest ^:parallel desugar-temporal-extract-test
+  (t/testing "desugaring :get-year, :get-month, etc"
+    (doseq [[op unit] mbql.u/temporal-extract-ops->unit]
+      (t/is (= [:temporal-extract [:field 1 nil] unit]
+               (mbql.u/desugar-temporal-extract [op [:field 1 nil]]))))))
+
```

Comment:
Would it make sense to test that de-sugaring works when these expressions are embedded deeper?